demo/pyproject.toml

```diff
@@ -0,0 +1,107 @@
+[tool.hatch.version]
+path = "src/__init__.py"
+allow-direct-references = true
+
+[project]
```

Comment:
Hmmm, do we need a `pyproject.toml` for the demo project?

Reply:
Yes. why not :)
it has more dependency comparing with real project. that's why i also used "local dependency" so any change you make in fastui you would see directly in demo. Would be very helpful for the development!